opentelemetry-semantic-conventions/src/opentelemetry/semconv/_incubating/attributes/aws_attributes.py

```diff
@@ -15,22 +15,22 @@
 
 from enum import Enum
 
-AWS_DYNAMODB_ATTRIBUTE_DEFINITIONS = "aws.dynamodb.attribute_definitions"
+AWS_DYNAMODB_ATTRIBUTE__DEFINITIONS = "aws.dynamodb.attribute_definitions"
```

Comment:
Why does the transformation turn `_` into `__` and not `.` into `__`? It seems to me like `__` is more of a separator than `_`. Is it just because there are fewer instances of `_` in the original semconv names?

Reply:
no strong opinion.

`__` is so ugly, I'd like to minimize number of it's occurrences and `.` is much more popular.
E.g. `HTTP_REQUEST_METHOD` looks better than `HTTP__REQUEST__METHOD` (to me)